great_expectations/experimental/datasource/fabric.py

```diff
@@ -114,7 +118,7 @@ def get_batch_list_from_batch_request(self, batch_request: BatchRequest) -> list
         )
 
         batch_metadata: BatchMetadata = self._get_batch_metadata_from_batch_request(
-            batch_request=batch_request
+            batch_request=batch_request, ignore_options=("dataframe",)
```

Comment:
I don't think this is strictly necessary but I also don't see this causing any problems.